.github/workflows/publish-image.yml

```diff
@@ -17,6 +17,12 @@ jobs:
       - name: Setup git lfs
         uses: ./.github/actions/setup-git-lfs
 
+      - name: Setup 3rd party dependencies
+        uses: ./.github/actions/setup-dependencies
+
+      - name: Poetry build
+        run: poetry build --format wheel
```

Comment:
```suggestion
        run: UNBLOB_BUILD_RUST_EXTENSION=1 poetry
```

Reply:
Ok, I am just checking this new `UNBLOB_BUILD_RUST_EXTENSION`. 

I just rebased it to main, I am adding this & doing some test run with the container publish.

Reply:
Cool, it is okay now, I felt Like I had to try it:
https://github.com/IoT-Inspector/unblob/runs/4533288685

I had to increase the minor version in the base image of the Dockerfile to work:
https://github.com/IoT-Inspector/unblob/pull/119/commits/67161481730b25adbdf449a8e25fc2868c1ed97a